File: aoj/volume100/csharp/A10028.cs
using System;
using System.Linq;

class AOJ
{
	static void Main()
	{
		Console.ReadLine(); // Skip 'n'
		var nums = Console.ReadLine().Split()
			.Select(s => int.Parse(s))
			.OrderBy(n => n)
			.Select(n => n.ToString())
			.ToArray();

		Console.WriteLine(string.Join(" ", nums));
	}
}

File: aoj/volume100/scala/A10028.scala
object Main {
  def main(args: Array[String]) {
    scala.io.StdIn.readLine // to skip n
    val nums = (scala.io.StdIn.readLine split " ").map(_.toInt)
    val sorted = nums.sorted
    val result = sorted.mkString(" ")
    println(result)
  }
}
